toasty/image.py

```diff
@@ -854,24 +842,44 @@ def asarray(self):
             self._array = np.asarray(self._pil)
         return self._array
 
-    def aspil(self):
+    def aspil(self, pixel_cut_low=None, pixel_cut_high=None):
```

Comment:
I think I would like to use a slightly different API here. I think that it is valuable for `aspil()` to behave as follows: it takes no parameters, and returns a PIL image if there is an exact expression of the image data in a form accepted by PIL. If that is not the case, it should raise an exception. This is all the intended current behavior.

I think we should add a different function, something like `coerce_into_pil()`, that provides the functionality you're adding here. The implementation will be similar, but I like having the API more clearly distinguish between cases where we expect the conversion to be fully automatic, and cases when we acknowledge that some kind of approximate transformation might have to happen to get the PIL form.